### ISSUE TITLE
docs: add English documentation

### ISSUE THREAD

**Issue by zhyncs** [baidu/babylon#57]
Hi Babylon genius, thank you for this amazing project. It would be more international if English documentation were available, as there are some English-speaking users. With ChatGPT, translating documents is easier than before. If possible, I hope the author can support this. Thank you! cc @oathdruid

**Comment by oathdruid** [comment]
Yes, it is a good idea. In fact I'm already trying to comment in english recently. I will find some time to do some docs translating, after finishing some coroutine support works right now.
